_data/simple-icons.json

```diff
@@ -1655,17 +1656,17 @@
         {
             "title": "Bitbucket",
             "hex": "0052CC",
-            "source": "https://www.atlassian.com/company/news/press-kit"
+            "source": "https://atlassian.design/resources/logo-library"
```

Comment:
I think we can also add https://atlassian.design/foundations/logos as guidelines